packages/core/test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -74,11 +74,15 @@ if (current.dialect.name === 'mssql') {
+      expectsql(this.queryGenerator.createTableQuery('myTable', { int: 'INTEGER COMMENT Foo Bar', varchar: 'VARCHAR(50) UNIQUE COMMENT Bar Foo' }, {}), { mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([int] INTEGER, [varchar] VARCHAR(50) UNIQUE); EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Foo Bar', @level0type = N'Schema', @level0name = N'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [int]; EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Bar Foo', @level0type = N'Schema', @level0name = N'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [varchar];` });
     });
 
     it('createTableQuery with comments and table object', function () {
-      expectsql(this.queryGenerator.createTableQuery({ tableName: 'myTable' }, { int: 'INTEGER COMMENT Foo Bar', varchar: 'VARCHAR(50) UNIQUE COMMENT Bar Foo' }, {}), { mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([int] INTEGER, [varchar] VARCHAR(50) UNIQUE); EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Foo Bar', @level0type = N'Schema', @level0name = 'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [int]; EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Bar Foo', @level0type = N'Schema', @level0name = 'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [varchar];` });
+      expectsql(this.queryGenerator.createTableQuery({ tableName: 'myTable' }, { int: 'INTEGER COMMENT Foo Bar', varchar: 'VARCHAR(50) UNIQUE COMMENT Bar Foo' }, {}), { mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([int] INTEGER, [varchar] VARCHAR(50) UNIQUE); EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Foo Bar', @level0type = N'Schema', @level0name = N'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [int]; EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Bar Foo', @level0type = N'Schema', @level0name = N'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [varchar];` });
+    });
+
+    it('createTableQuery with comments,  table object and schema', function () {
```

Comment:
Can be in a future PR due to how we're testing `createTableQuery` instead of `commentTemplate` but adding a test where we set the schema globally would be nice. For example you can see the TypeScript test files for each function in the query generator unit test folder

Reply:
It looks like the test code doesn't have a huge amount of coverage right now for anything using non-standard schemas. I think you are right that this can be a future PR or get added on the to-do list. For now I'm happy to throw a couple more schema checks e.g. in create table and create column queries if need be.

Reply:
Yeah, it's something we're slowly adding. Let's just do it in a future PR so we can get this merged once these tests have been moved to `test/unit/query-generator/create-table-query.test.ts`

Reply:
@WikiRik @ephys I have added in the unit tests under query-generator - i had to create the create-table-query.test.ts file as I couldn't see it in the repo. I have added in the same tests as I was using but have defined them for the other dialects, so the unit tests complete. I've also reverted the extra schema test in the mssql dialect unit test.

Reply:
Yes, I thought we already had the file but I'm creating it in https://github.com/sequelize/sequelize/pull/15526
Maybe I can clean up the unit tests there first and we merge this after? If you're fine with that

Reply:
@WikiRik Whichever is easiest, no hurry on my end.

Reply:
I've cleaned up the unit tests there, just waiting for reviews